components/list/list.js

```diff
@@ -64,8 +64,6 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 		this.dragMultiple = false;
 		this.extendSeparators = false;
 		this.grid = false;
-		this._itemsShowingCount = 0;
-		this._itemsShowingTotalCount = 0;
```

Comment:
I talked with @dbatiste about the difference between these two variables, and we agreed that the distinction is no longer necessary. `_itemsShowingTotalCount` originally had a more expensive calculation that recursed through nested lists, but more recently it was decided that we didn't want to show that detail, and only want the "showing count" to represent top-level items.

The expensive calcs were [removed as part of expand/collapse](https://github.com/BrightspaceUI/core/pull/3046/files#diff-afabf94fa43a55055dff6b1ca27abd24b629f017ad9dee5d4c48f123e80a0533L194), so following that change the two variables were essentially identical.